LICENSE

```diff
@@ -0,0 +1,15 @@
+The Juju GUI is the graphical user interface used to interact with Juju.
+Copyright (C) 2017  Canonical Ltd.
```

Comment:
It it's the license file for the whole program, the (C) should probably be when it was started?

Dunno. Maybe the (C) is just for the file?